nuget/spec/dependabot/nuget/update_checker/nupkg_fetcher_spec.rb

```diff
@@ -135,7 +64,7 @@
           )
       end
 
-      it { is_expected.to eq("https://pkgs.dev.azure.com/dnceng/public/_apis/packaging/feeds/dotnet-public/nuget/packages/Newtonsoft.Json/versions/13.0.1/content?sourceProtocolVersion=nuget&api-version=7.0-preview") }
+      it { is_expected.to eq("https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/45bacae2-5efb-47c8-91e5-8ec20c22b4f8/nuget/v3/flat2/newtonsoft.json/13.0.1/newtonsoft.json.13.0.1.nupkg") }
```

Comment:
The magic GUIDs in this URL come from the `index.json` that this test returns a few lines up.